observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| availabilityTime \[@value]                                               | same as the Parent Compound Statement's `availatilityTime`                                                                                                                                                                                             |
+| value                                                                    | `Observation.component[index].valueQuantity`                                                                                                                                                                                                           |
+| referenceRange / referenceInterpretationRange / text                     | `Observation.component[index].referenceRange.text`                                                                                                                                                                                                     |
+| referenceRange / referenceInterpretationRang / value                     | `Observation.component[index].referenceRange.value`                                                                                                                                                                                                    |
+
+### Narrative Statement components
+
+A blood pressure may have a `NarrativeStatement` component if the parent observation has a comment or if `Observation.component.dataAbsentReason`, 
+`Observation.bodySite` or `Observation.component.interpretation` are populated.   
+
+| Mapped to (XML HL7 CompoundStatement / component / NarrativeStatement) | Mapped from (JSON FHIR / other source )                                                                                                              |
+|------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                                                     | Unique ID generated by the adaptor                                                                                                                   |
+| text                                                                   | Concatenated from `Observation.comment`, `Observation.bodySite`, `Observation.component.dataAbsentReason` and `Observation.component.interpretation` |
+| statusCode                                                             | fixed value = `"COMPLETE"`                                                                                                                           |
```

Comment:
I think in Narrative statement there is also a participant field which is missed here. Also, I couldn't find the "status code" (documented in here) in the codebase.

Reply:
It's a bit confusing the participant block is at the end of the template, but because it's not surrounded by `<component` it's part of the `compondStatement`. The `statusCode` is hardcoded in the template too. The template is ` ehr_compound_statement_blood_pressure_template.mustache`